native/level2double.go

```diff
@@ -58,21 +58,22 @@ var _ blas.Float64Level2 = Implementation{}
 */
 
 const (
-	mLT0         string = "referenceblas: m < 0"
-	nLT0         string = "referenceblas: n < 0"
-	kLT0         string = "referenceblas: k < 0"
-	badUplo      string = "referenceblas: illegal triangularization"
-	badTranspose string = "referenceblas: illegal transpose"
-	badDiag      string = "referenceblas: illegal diag"
-	badSide      string = "referenceblas: illegal side"
-	badLdaRow    string = "lda must be greater than max(1,n) for row major"
-	badLdaCol    string = "lda must be greater than max(1,m) for col major"
-	badLda       string = "lda must be greater than max(1,n)"
-
-	badLdaTriBand string = "goblas: lda must be greater than k + 1 for general banded"
-	badLdaGenBand string = "goblas: lda must be greater than 1 + kL + kU for general banded"
-	kLLT0         string = "goblas: kL < 0"
-	kULT0         string = "goblas: kU < 0"
+	mLT0         = "blas: m < 0"
+	nLT0         = "blas: n < 0"
+	kLT0         = "blas: k < 0"
+	badUplo      = "blas: illegal triangle"
+	badTranspose = "blas: illegal transpose"
+	badDiag      = "blas: illegal diagonal"
+	badSide      = "blas: illegal side"
+
+	badLdA = "blas: index of a out of range"
+	badLdB = "blas: index of b out of range"
+	badLdC = "blas: index of c out of range"
+
+	badLdaTriBand = badLdA // was: "blas: lda must be greater than k + 1 for general banded"
```

Comment:
Yes, they are a reminder that those messages give more information.

Are you happy to lose that information from the message? If so, I'll drop them, otherwise I can add similar messages to the appropriate places in blas/cgo.